sys/include/vfs.h

```diff
@@ -141,11 +141,11 @@ extern "C" {
 
 #  if (__SIZEOF_POINTER__ == 8)
 #    define FATFS_VFS_DIR_BUFFER_SIZE      (64 + _FATFS_DIR_LFN + _FATFS_DIR_EXFAT)
-#    define FATFS_VFS_FILE_BUFFER_SIZE     (57 + VFS_NAME_MAX + _FATFS_FILE_CACHE + \
+#    define FATFS_VFS_FILE_BUFFER_SIZE     (64 + VFS_NAME_MAX + _FATFS_FILE_CACHE + \
                                            _FATFS_FILE_SEEK_PTR + _FATFS_FILE_EXFAT)
 #  else
 #    define FATFS_VFS_DIR_BUFFER_SIZE      (44 + _FATFS_DIR_LFN + _FATFS_DIR_EXFAT)
-#    define FATFS_VFS_FILE_BUFFER_SIZE     (41 + VFS_NAME_MAX + _FATFS_FILE_CACHE + \
+#    define FATFS_VFS_FILE_BUFFER_SIZE     (48 + VFS_NAME_MAX + _FATFS_FILE_CACHE + \
```

Comment:
Looks like some padding thing, rounding up to the next full word seems to be sufficient though

```suggestion
#    define FATFS_VFS_FILE_BUFFER_SIZE     (44 + VFS_NAME_MAX + _FATFS_FILE_CACHE + \
```

Reply:
indeed 44 seems also to be sufficient also for values of VFS_NAME_MAX of up to 56 which I tested again

Reply:
I wonder why I put 48 here before